test/e2e/operator_groups_e2e_test.go

```diff
@@ -274,9 +316,78 @@ func TestOperatorGroup(t *testing.T) {
 		return false, nil
 	})
 
+	// check rbac in target namespace
+	informerFactory := informers.NewSharedInformerFactory(c.KubernetesInterface(), 1*time.Second)
+	roleInformer := informerFactory.Rbac().V1().Roles()
+	roleBindingInformer := informerFactory.Rbac().V1().RoleBindings()
+	clusterRoleInformer := informerFactory.Rbac().V1().ClusterRoles()
+	clusterRoleBindingInformer := informerFactory.Rbac().V1().ClusterRoleBindings()
+
+	// kick off informers
+	stopCh := make(chan struct{})
+	defer func() {
+		stopCh <- struct{}{}
+		return
+	}()
+
+	for _, informer := range []cache.SharedIndexInformer{roleInformer.Informer(), roleBindingInformer.Informer(), clusterRoleInformer.Informer(), clusterRoleBindingInformer.Informer()} {
+		go informer.Run(stopCh)
+
+		synced := func() (bool, error) {
+			return informer.HasSynced(), nil
+		}
+
+		// wait until the informer has synced to continue
+		err := wait.PollUntil(500*time.Millisecond, synced, stopCh)
+		require.NoError(t, err)
+	}
+
+	ruleChecker := install.NewCSVRuleChecker(roleInformer.Lister(), roleBindingInformer.Lister(), clusterRoleInformer.Lister(), clusterRoleBindingInformer.Lister(), &aCSV)
+
+	t.Log("Waiting for operator to have rbac in target namespace")
+	err = wait.Poll(pollInterval, pollDuration, func() (bool, error) {
+		for _, perm := range permissions {
+			sa, err := c.GetServiceAccount(opGroupNamespace, perm.ServiceAccountName)
+			require.NoError(t, err)
+			for _, rule := range perm.Rules {
+				satisfied, err := ruleChecker.RuleSatisfied(sa, otherNamespaceName, rule)
```

Comment:
added a test to verify RBAC in target namespaces (kinda funky with informers in the test, but it works)